distributionviewer/core/static/js/app/components/layouts/main-layout.js

```diff
@@ -11,19 +11,27 @@ export default function(props) {
       <header className="primary-header">
         <div>
           <h1>
-            <Link className="primary-header-link" to="/">Firefox Distribution Viewer</Link>
+            <Link className="primary-header-link" to={'/?pop=' + props.whitelistedPopulations}>Firefox Distribution Viewer</Link>
```

Comment:
In *router.js*, I made *?pop=All* the default if no other *?pop* options are set. But to be safe I can add *All* as the default in *app-container.js*.